examples/ads.amp.html

```diff
@@ -1199,6 +1200,34 @@ <h2>Outbrain widget</h2>
       data-testMode="true">
   </amp-embed>
 
+  <h2>Pixels Examples</h2>
+  <amp-ad width="300" height="250"
+          type="pixels"
```

Comment:
can you match indentation of other examples

Reply:
Updated.